### ISSUE TITLE
[API View] Show reference docs for specific method/class

### ISSUE THREAD

**Issue by srnagar** [Azure/azure-sdk-tools#2389]
In Java API view, there's an option to enable docs view which shows the JavaDocs for every class/method. It would be great if we can support viewing/hiding reference docs of only a specific method/class of interest rather than showing docs for all which makes the view too large.

(Adding Java label since only Java API views have documentation included today but would be useful to do the same when documentation is included for other languages)

**Comment by kurtzeborn** [comment]
Since this is a Java-specific request, leaving it labeled "Client" for the Java team.  Please work with @praveenkuttappan to get help/support for integrating this into Apiview.

**Comment by JonathanGiles** [comment]
This isn't a Java-specific request - the same feature applies across languages and needs to be implemented in APIViewWeb, rather than anything specifically to the Java parser.

**Comment by JonathanGiles** [comment]
@srnagar It would be good to get your thoughts on how this should be implemented. Do you imagine there would be a clickable button near where the + button is for comments, that would enable users to expand the docs for that page? I can't imagine any other way to toggle the docs on/off that wouldn't be annoying.

**Comment by srnagar** [comment]
It would be good to have a "show docs" icon next to the `+` icon we have for adding comments. When the icon is clicked, the docs pertaining to the corresponding class/method should show and when the icon is clicked again, the docs should be hidden. 

This brings up a question on what to do with the line numbers when the docs are displayed/hidden. My suggestion would be to not have line numbers for the docs (when displayed) as that will require updating the line numbers for the entire API view.

![image](https://user-images.githubusercontent.com/51379715/181386584-83fb9cdc-1603-42af-aa79-c3def6e1a102.png)

**Comment by JonathanGiles** [comment]
If this change is made, it is going to result in a change to the existing behaviour where all reference documentation can be turned on or off for a single review. This is because in the current code, the reference documentation is not loaded into the HTML sent to the client, unless documentation is set to show. In this new design, we are going to have to send all HTML to the client, with the documentation show button (for a single API) calling some JavaScript to toggle the visibility of that specific reference documentation. The existing toggle to hide / show all reference documentation will then need updating such that it toggles the visibility of the reference documentation without needing to reload from the server.

In short, we move the burden of toggling the visibility from server side to client side, but we put more burden on the server to send down HTML with all reference documentation in all cases.

**Comment by Vailorium** [comment]
@JonathanGiles
